### ISSUE TITLE
Run package to token code file conversion outside APIView server

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-tools#3596]
Currently APIView runs a process within server to convert each language specific package to token file. This can cause security issue if language specific parser uses any vulnerable third party dependencies. Conversion to token file will be completed by CI pipeline for PR and CI runs and send details to APIView to pull these files from devops pipeline. Manual review generation will offload conversion process to Azure devops pipeline. This approach will allow us to support more languages in our approval gate checking as part of release. For e.g. Swift requires specific thirdparty tools and running environment that's currently not available in APIView. New offloaded approach will allow language to use any specific environment it wants.

**Comment by JonathanGiles** [comment]
Can I ask that rather than remove this functionality that you put it behind a configuration flag, so that #3564 remains possible for use cases where local generation of token files is acceptable?

**Comment by praveenkuttappan** [comment]
Yes, We will have config param to enable this explicitly so by default local development will not be impacted.

**Comment by JonathanGiles** [comment]
> Yes, We will have config param to enable this explicitly so by default local development will not be impacted.

Even beyond local development, I foresee a future where external teams may choose to deploy APIView and operate it in its current mode of direct upload, rather than using a pipeline to do the parsing. It would be nice to have that continue to be an option, so thanks for making it configurable.

**Comment by maririos** [comment]
@praveenkuttappan Is this the same as the Sandboxing project? should we close this and leave the individual language issues?

**Comment by maririos** [comment]
We’ve decided to prioritize APIView server changes related to Copilot work.
